### PR TITLE
Updates for 16.04 packaging

### PR DESCRIPTION
**Pull request by akva2**
- install ebos
- update debian packaging

please review and all that but do not merge until i have added the rhel packaging (opening pr early to get the installation part reviewed).

@andlaus as requested.

### CODE REVIEW

**Comment by akva2** [comment]
redhat packaging added.

File: debian/control [hunk 1]
```diff
@@ -40,3 +40,9 @@ Description: documentation for ewoms
  framework which is capable of capturing all macro-scale scenarios
  relevant for academic research and industrial applications involving
  flow and transport processes in porous media.
+
+Package: ebos
+Section: science
+Architecture: any
+Description: ebos is an ECL simulator
+ ebos is an ECL simulator.
```
Comment (andlaus) [inline debian/control]: stupid question: which files are included into that package?

Comment (akva2) [inline debian/control]: the stuff in ebos.install

Comment (akva2) [inline debian/control]: globbing removed.

Comment (andlaus) [inline debian/control]: ah, ok. sorry for such stupid questions, I'm not really into the packaging business.

as far as I can see, the PR can be merged. please press Green yourself once you deem it ready.

Comment (akva2) [inline debian/control]: no worries. one has to learn at some point. as long as there is no external files needed to execute ebos...

**Pull request merged**